src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+class Task {
```

Comment:
Same as the question in `Duke.java`. Perhaps we can put classes in a package `exception` etc.